mm2src/crypto/src/encrypt.rs

```diff
@@ -107,6 +113,7 @@ pub fn encrypt_data(
     let tag = mac.finalize().into_bytes();
 
     let encrypted_data = EncryptedData {
+        version: ENCRYPTED_DATA_VERSION,
```

Comment:
Q: is this a breaking change?

Reply:
Not really as this has not been used by anyone yet.

Reply:
Yeah that's why I wonder, if we use these values to encrypt something, previously encrypted values will be mismatched.